job_templates/service_action_-_enable_web_console.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+name: Service Action - Enable Web Console
+job_category: Ansible Services
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+---
+- hosts: all
+  tasks:
+  - name: ensure cockpit is installed
+    package:
+      name: "cockpit"
+      state: present
+
+  - name: enable firewall rule for cockpit
+    block:
+      - name: install firewalld bindings
+        package:
+          name: python-firewall
```

Comment:
Various distros have started to use `python2-` or `python3-` prefixes. Does this work on EL8 for example?